records_mover/db/postgres/sqlalchemy_postgres_copy.pyi

```diff
@@ -1,17 +1,18 @@
-from typing import Union, IO
+from typing import Union, IO, Optional
 import sqlalchemy
 
 
 def copy_from(source: IO[bytes],
               dest: sqlalchemy.schema.Table,
-              engine_or_conn: Union[sqlalchemy.engine.Engine, sqlalchemy.engine.Connection],
+              engine_or_conn: Optional[Union[sqlalchemy.engine.Engine,
+                                             sqlalchemy.engine.Connection]],
```

Comment:
why do these change to optional? How are you going to copy if you dont have a connection or an engine? I know in some of the upper code that stores these objects theyre passed around as optional, but shouldnt you have to unwrap the option at some point before you actually do anything with it?

Reply:
I see no reason for it to be Optional so removing that